pkg/antctl/command_definition.go

```diff
@@ -72,6 +75,7 @@ const (
 const (
 	flat commandGroup = iota
 	get
+	query
```

Comment:
This is true. We made the design decision to group query commands into their own category because of their difference in behavior from the perspective of the user. Additionally, query commands have a different output transform (they return a list of tables as opposed to single tables as in get commands). See constructTableOutputForQueryCommand in command_definition.go.

Reply:
OK, we could see if it's possible to find a way to unify them later.